packages/stark-ui/src/modules/table/components/dialogs/multisort.component.html

```diff
@@ -14,7 +14,7 @@ <h1 mat-dialog-title translate>STARK.MULTI_COLUMN_SORTING.TITLE</h1>
 					<mat-select [value]="rule" (selectionChange)="onColumnChange(rule, $event.value)">
 						<ng-container *ngFor="let ruleOption of rules; trackBy: trackRuleFn">
 							<mat-option *ngIf="ruleOption == rule || !ruleOption.sortDirection" [value]="ruleOption">
-								{{ (ruleOption.column.label ? ruleOption.column.label : ruleOption.column.name) | translate }}
+								{{ (ruleOption.column.headerLabel ? ruleOption.column.headerLabel : ruleOption.column.name) | translate }}
```

Comment:
This could be simplified the code behind `headerLabel` in "column.component.ts" is:
```typescript
public get headerLabel(): string {
	return this._headerLabel || this.name;
}
``` 

What do you think of directly put:
```html
{{ ruleOption.column.headerLabel | translate }}
```
since it has exactly the same result and it's simplier ? 😊

Reply:
yes sure

Reply:
done